components/org.wso2.carbon.identity.api.server.userstore/org.wso2.carbon.identity.api.server.userstore.v1/src/main/java/org/wso2/carbon/identity/api/server/userstore/v1/core/ServerUserStoreService.java

```diff
@@ -403,6 +437,55 @@ public UserStoreResponse patchUserStore(String domainId, List<PatchDocument> pat
         return null;
     }
 
+    /**
+     * To make a partial update single or multiple claim attribute mappings of a specific user store.
+     *
+     * @param userstoreDomainId     user store domain id
+     * @param claimAttributeMapping list of claim attribute mappings in patch request
+     */
```

Comment:
Let's add `.` at the end of each sentence.

Reply:
Addressed with commit c731518fd0ea44d2d06c025d044e61114470